src/components/networked-video-player.js

```diff
@@ -21,15 +19,13 @@ AFRAME.registerComponent("networked-video-player", {
 
     await nafConnected();
 
-    const networkedEl = NAF.utils.getNetworkedEntity(this.el);
+    const networkedEl = await NAF.utils.getNetworkedEntity(this.el);
     if (!networkedEl) {
-      throw new Error(
-        "Video player must be added on a node, or a child of a node, with the `networked` component."
-      );
+      throw new Error("Video player must be added on a node, or a child of a node, with the `networked` component.");
     }
 
     const ownerId = networkedEl.components.networked.data.owner;
-    const stream = await NAF.connection.adapter.getMediaStream(ownerId);
+    const stream = await NAF.connection.adapter.getMediaStream(ownerId, "video");
```

Comment:
Hmm, I don't think there is a second argument to `getMediaStream`. https://github.com/mozilla/naf-janus-adapter/blob/master/src/index.js#L351

Reply:
Yup, added it for naf-janus-adapter here: https://github.com/mozilla/naf-janus-adapter/pull/29

Not sure if it's worth suggesting it for all networked-aframe adapters.